### DISCUSSION TITLE
Access the portal site using a different domain name

### DISCUSSION THREAD

**Discussion by sergeByishimo** [exoplatform/exo-community#33] [Q&A]
Hi! I saw the server `exo.base.url` used to configure the base URL that users use to access eXo Platform, and I'm wondering if its possible to access the portal or a specific portal site using a different domain name other than the  server `exo.base.url` ??

**Comment by rdenarie** [comment]
Hello
Short answser is : no, it is not possible.

Long answer : 
In fact exo.base.url is used to build some urls which are provided by the users. 

You will be able to have different domain names pointing on the same eXo server at DNS level, but eXo Server will always build his urls with the domain provided in the property.
So users will switch from one domain to another during navigation, and it will not work as you want.
